Flagsmith.FlagsmithClient/AnalyticFlag.cs

```diff
@@ -0,0 +1,45 @@
+﻿using FlagsmithEngine.Feature.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using Newtonsoft.Json;
+using System.Threading.Tasks;
+
+namespace Flagsmith
+{
+    public class AnalyticFlag : Flag
```

Comment:
I'm not sure why we need this separate class? Couldn't this logic be contained within the regular `Flag` class?

Reply:
removed analyticFlag class , and updated its logic like python implementation.